packages/client/src/app/pages/work-page/work-page.component.ts

```diff
@@ -105,6 +107,13 @@ export class WorkPageComponent implements OnInit {
     }, () => {
       this.loading = false;
     });
+
+    this.route.queryParamMap.subscribe(queryParams => {
```

Comment:
If you're just getting query params once on Init, you can probably get away with `queryParamMap.snapshot` so you don't have to wait for the observable.

Reply:
I'm unable to get it working outside of the subscription. :<

Reply:
really? huh, I'll poke it once changes land in master